### PR TITLE
add tooltip log viewer times button

### PR DESCRIPTION
**Pull request by gimler**

### CODE REVIEW

**Comment by gimler** [comment]
cla signed

File: src/app/frontend/logs/logs.html [hunk 1]
```diff
@@ -74,6 +74,10 @@
       <md-button class="kd-logs-toolbar-button"
                  ng-click="ctrl.onShowTimestamp()">
         <md-icon class="kd-logs-icon">{{ctrl.logsService.getShowTimestamp() ? 'timer' : 'timer_off'}}</md-icon>
+        <md-tooltip md-delay="500"
+                    md-autohide>
+          [[Toggle times|Tooltip for button that toggles times option in the log viewer.]]
```
Comment (stefansundin) [inline src/app/frontend/logs/logs.html]:
```suggestion
          [[Toggle timestamps|Tooltip for button that toggles timestamps in the log viewer.]]
```

**Comment by maciaszczykm** [comment]
LGTM but translations are missing. Generate them using guide from wikis.

**Comment by maciaszczykm** [comment]
Please reopen after performing requested changes.

/close

**Pull request closed without merging**